Makefile

```diff
@@ -1,7 +1,17 @@
-REGISTRY       ?= quay.io
-ORG            ?= kubernetes_incubator
-TAG            ?= $(shell git describe --always --dirty)
-IMAGE          ?= $(REGISTRY)/$(ORG)/node-feature-discovery-operator:$(TAG)
+IMAGE_BUILD_CMD ?= docker build
+IMAGE_BUILD_EXTRA_OPTS ?=
+IMAGE_PUSH_CMD ?= docker push
+
+VERSION := $(shell git describe --tags --dirty --always)
+
+IMAGE_REGISTRY := k8s.gcr.io/nfd
+IMAGE_NAME := node-feature-discovery-operator
+IMAGE_TAG_NAME := $(VERSION)
+IMAGE_EXTRA_TAG_NAMES :=
+IMAGE_REPO := $(IMAGE_REGISTRY)/$(IMAGE_NAME)
+IMAGE_TAG := $(IMAGE_REPO):$(IMAGE_TAG_NAME)
+IMAGE_EXTRA_TAGS := $(foreach tag,$(IMAGE_EXTRA_TAG_NAMES),$(IMAGE_REPO):$(tag))
```

Comment:
I think we could use `?=` for most (if not all) of these – probably `IMAGE_REPO` and `IMAGE_NAME` could be considered "internal" variables and initialized with `:=`. That way, we could drop the `-e` make option from the test-infra scripts.

Reply:
agree , mostly for testing, you want to override those for a personal repo